docs/issues-and-help.txt

```diff
@@ -0,0 +1,55 @@
+.. _laravel-issues-and-help:
+
+=============
+Issues & Help
+=============
+
+We are lucky to have a vibrant PHP community that includes users of varying
+experience with MongoDB PHP Library and {+odm-short+}. To get support for
+general questions, search or post in the
+:community-forum:`MongoDB Community Forums <>`.
```

Comment:
Q: can this link be more specific to the library? like is there a PHP tag or something

Reply:
yes there is - updated the link!